### ISSUE TITLE
Force a new cluster to bootstrap from a wal-e backup

### ISSUE THREAD

**Issue by conord33** [zalando/spilo#216]
If I have made a back up with wal-e in my patroni cluster and delete the whole cluster (including the kubernetes endpoint containing the `initialize` key) patroni will bootstrap a new cluster and ignore the previous backup made with wal-e. If I delete everything except the kubernetes endpoint containing the `initialize` key it will correctly bootstrap from the backup on wal-e. 

Is there a way to force patroni to restore from the wal-e backup even if the `initialize` key does not exist? Is there a way to tell patroni to bootstrap from a specific point in the wal-e backup? (PITR). 

Sorry if I missed this in the docs or readme!

**Comment by conord33** [comment]
Thanks for the quick reply! I will try it out and let you know if it works or not.

**Comment by conord33** [comment]
Okay so I got it to work but had to manually set the `CLONE_WALE_ENV_DIR` field. (I am using GCS by the way)

```
CLONE_METHOD=CLONE_WITH_WALE
CLONE_WALE_ENV_DIR=/home/postgres/etc/wal-e.d/env
CLONE_WAL_GSC_BUCKET=bucket
WAL_GCS_BUCKET=bucket
```

Without setting `CLONE_WALE_ENV_DIR` manually it will use `/home/postgres/etc/wal-e.d/env-clone-patroni` which is empty. And causes the the `/clone_with_s3.py` script to fail.

If I set it to `WALE_ENV_DIR` it works, but will also push a backup to the same bucket. Is there a way to configure it to clone from one bucket and either not use wal_e after the initial clone or backup to another bucket? This could be useful for spinning up a new cluster to test the data of another with out pushing backups or wall files to the one you cloned from.
